core/document/src/document.rs

```diff
@@ -41,7 +41,10 @@ impl Document {
 			return;
 		}
 		if path.as_slice() == self.work_mount_path {
+			// TODO: Handle if mounted in nested folders
```

Comment:
Shouldn't this already work?

Reply:
```
<folder scale = 2>
    <folder scale = 3> <--- Mounted here
    </folder>
</folder>
```
@TrueDoctor  with the above, the work folder should have a scale of 6, but this is not handled currently.

Reply:
That makes sense, I thought it would just inherit the scale from the folder it is mounted in -> have a scale of two. But I'll have to review the full pr to understand the details